hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/blockmanagement/TestDatanodeManager.java

```diff
@@ -1126,4 +1128,66 @@ public MockDfsNetworkTopology(){
       super();
     }
   }
+
+  @Test
+  public void testComputeReconstructedTaskNum() throws IOException {
+    verifyComputeReconstructedTaskNum(100, 100, 150, 250, 100);
+    verifyComputeReconstructedTaskNum(200, 100000, 200000, 300000, 400000);
+    verifyComputeReconstructedTaskNum(1000000, 100, 150, 250, 100);
+    verifyComputeReconstructedTaskNum(14000000,200, 200, 400, 200);
```

Comment:
checkstyle issue, `verifyComputeReconstructedTaskNum(14000000, 200, 200, 400, 200);`

Reply:
Thanks for the guidance! I will expedite the code revisions accordingly.